roles/kustomize_deploy/tasks/execute_step.yml

```diff
@@ -60,6 +60,10 @@
       vars:
         hooks: "{{ stage.pre_stage_run }}"
         step: "pre_stage_{{ stage_id }}_run"
+        _arch_repo_dir: >-
```

Comment:
hmmm it shouldn't be needed actually.
Hooks are injecting the environment in the playbook:
https://github.com/openstack-k8s-operators/ci-framework/blob/ba36715ce8cc58bbacd937c9b21ae1c71cc638d8/roles/run_hook/tasks/playbook.yml#L46-L62

So anything passed to the initial playbook will end in the env passed down to the hook play, without any other needs.

Reply:
Namely, you should be able to use this top-level parameter as-is in the hooked playbook:
`docs/source/usage/01_usage.md:- `cifmw_architecture_repo`: (String) Path of the architecture repository on the controller node.`